packages/dx-react-grid/src/plugins/table-header-row.jsx

```diff
@@ -41,6 +42,7 @@ export class TableHeaderRow extends React.PureComponent {
           { name: 'TableColumnResizing', optional: true },
         ]}
       >
+        <Getter name="isRtl" computed={isRtlComputed} />
```

Comment:
It isn't required to define such Getter in each plugin. You can use the previously calculated value in TemplateConnector directly.